src/main/java/core/basesyntax/service/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void writeFile(String report, String filePath) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            writer.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write a " + filePath, e);
```

Comment:
Consider providing more specific information in the exception message, such as the reason why the file could not be written (e.g., permissions issue, disk space), to aid in debugging.